build-uboot-rpi64.sh

```diff
@@ -52,7 +52,7 @@ export CROSS_COMPILE=aarch64-linux-gnu-
 export ARCH=aarch64
 
 # Test if the toolchain is actually installed
-aarch64-linux-gnu-gcc --version
+${CROSS_COMPILE}gcc --version
```

Comment:
This should be part of the previous commit.

This Pull Request introduces this script for the first time, hence it does not have any historical value to look at how it was first introduced and then modified the --version command to use the `CROSS_COMPILE` variable. -> This is my reasoning for whatever happened in this script should be self-contained in one commit.

Reply:
Folded it into the mentioned commit.